app/src/main/java/de/xikolo/controllers/helper/ShortcutHelper.kt

```diff
@@ -0,0 +1,62 @@
+package de.xikolo.controllers.helper
+
+import android.annotation.TargetApi
+import android.content.Context
+import android.content.Intent
+import android.content.pm.ShortcutInfo
+import android.content.pm.ShortcutManager
+import android.graphics.drawable.Icon
+import android.os.Build
+import androidx.core.content.ContextCompat
+import androidx.core.content.ContextCompat.getSystemService
+import de.xikolo.R
+import de.xikolo.controllers.course.CourseActivityAutoBundle
+import de.xikolo.storages.RecentCoursesStorage
+
+const val MAX_SHORTCUTS = 4
+
+class ShortcutHelper {
```

Comment:
Could be an `object` as well. Alternatively you could add the `Context` as a constructor parameter. I would go with object.